src/index.html

```diff
@@ -1,13 +1,44 @@
-<!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+  <!doctype html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet"
+          href="./style.css">
+    <link rel="stylesheet"
+          href="./normalize.css">
+    <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
+  </head>
+  <body class="product">
+    <div class="product__card"
+         data-qa="card"
+    >
+      <img src="/images/imac.jpeg"
+           alt="imac"
+            class="product__card-photo"
+      >
```

Comment:
так атрибути ми не верівнюємо
```suggestion
    <div class="product__card" data-qa="card">
      <img
        src="/images/imac.jpeg"
        alt="imac"
        class="product__card-photo"
      >
```